packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -18,8 +18,12 @@ import 'layout_helper.dart';
 import 'object.dart';
 import 'selection.dart';
 
-/// The start and end positions for a word.
-typedef _WordBoundaryRecord = ({TextPosition wordStart, TextPosition wordEnd});
+/// The start and end positions for a text boundary.
+typedef _TextBoundaryRecord = ({TextPosition boundaryStart, TextPosition boundaryEnd});
```

Comment:
I just learned that it's possible to typedef a record like this, pretty cool 👍 .  Though I guess I'm out of date since this file was doing it before this PR 😂